### PR TITLE
[Docs] Fix CLI docs

### PR DESCRIPTION
**Pull request by alonmr**
https://jira.iguazeng.com/browse/ML-4594

### CODE REVIEW

File: docs/cli.md [hunk 1]
```diff
@@ -67,18 +68,18 @@ Examples:
...
+|--------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| &minus;&minus;kind &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; | Clean resources for all runs of a specific kind (e.g. job).                                                                                                                                                                                                          |
+| &minus;&minus;id &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;   | Delete the resources of the mlrun object twith this identifier. For most function runtimes, runtime resources are per Run, and the identifier is the Run’s UID. For DASK runtime, the runtime resources are per Function, and the identifier is the Function’s name. |
```
Comment (jillnogold) [inline docs/cli.md:74]: | &minus;&minus;id &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; | Delete the resources of the mlrun object with this identifier. For most function runtimes, runtime resources are per run, and the identifier is the run’s UID. For DASK runtime, the runtime resources are per function, and the identifier is the function’s name. |

File: docs/cli.md [hunk 2]
```diff
@@ -67,18 +68,18 @@ Examples:
...
+| Flag                                                         | Description                                                                                                                                                                                                                                                          |
+|--------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| &minus;&minus;kind &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; | Clean resources for all runs of a specific kind (e.g. job).                                                                                                                                                                                                          |
```
Comment (jillnogold) [inline docs/cli.md:73]: is this clean or clear?

File: docs/cli.md [hunk 3]
```diff
@@ -67,18 +68,18 @@ Examples:
...
-|  -f, &minus;&minus;force   |  Delete the runtime resource even if they're not in terminal state or if the grace period didn’t pass.|
-|  -gp, &minus;&minus;grace-period &nbsp; &nbsp; &nbsp;  &nbsp; &nbsp; &nbsp; | Grace period, in seconds, given to the runtime resource before they are actually removed, counted from the moment they moved to the terminal state.  |
+| Options                                       | Description                                                                                                                                         |
+|-----------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------|
+| &minus;&minus;api                             | URL of the mlrun-api service.                                                                                                                       |
+| &minus;ls, &minus;&minus;label&minus;selector | Delete only runtime resources matching the label selector.                                                                                          |
```
Comment (jillnogold) [inline docs/cli.md:80]: | &minus;ls, &minus;&minus;label&minus;selector | Delete only the runtime resources matching the label selector.

File: docs/cli.md [hunk 4]
```diff
@@ -176,38 +195,54 @@ Examples:
...
+| &minus;&minus;origin&minus;file                | For internal use                                                                                        |
+| &minus;&minus;label                            | Run labels (key=val)                                                                                    |
+| -w, &minus;&minus;watch                        | Watch/tail run log                                                                                      |
+| &minus;&minus;verbose                          | Verbose log                                                                                             |
+| &minus;&minus;scrape&minus;metrics             | Whether to add the `mlrun/scrape-metrics` label to this run's resources                                 |
+| &minus;&minus;env&minus;file                   | Path to .env file to load config/variables from                                                         |
+| &minus;&minus;auto&minus;build                 | When set functions will be built prior to run if needed                                                 |
```
Comment (jillnogold) [inline docs/cli.md:237]: | &minus;&minus;auto&minus;build | When set, the function image will be built prior to run if needed

File: docs/cli.md (hunk 2 above)
Comment (alonmr) [reply docs/cli.md:73]: I think maybe clear is the better word here but the CLI command is called `clean` so prefer to keep it consistent. Difference sounds minor to me

**Pull request merged**